### PR TITLE
Bump NPM dependencies and switch to ESLint

### LINKED ISSUE #1220: Use ESLint instead of JSCS in our own build

### PR DESCRIPTION
**Pull request by adidahiya**
fixes #1220

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,21 +22,21 @@
   "dependencies": {
     "colors": "^1.1.2",
     "diff": "^2.2.1",
-    "findup-sync": "~0.3.0",
-    "glob": "^6.0.1",
+    "findup-sync": "~0.4.0",
```
Comment (JKillian) [inline package.json]: noooo undo!  (see https://github.com/palantir/tslint/pull/1108)

**Comment by JKillian** [comment]
Also, build errors because of invalid peer dependencies :/

image

**Comment by adidahiya** [comment]
yep, noticed the invalid peer deps... will try to investigate tomorrow

**Comment by JKillian** [comment]
Related: https://github.com/unindented/grunt-npm-command/issues/7
I think that might solve the `peerDependency` issues

**Comment by adidahiya** [comment]
I'd actually prefer to remove the grunt-npm-command dependency. It is a small package with very few users.

**Comment by JKillian** [comment]
Sounds good, as long as things work correctly across platforms/CIs. Note https://github.com/palantir/tslint/pull/1172 which is the relevant discussion when it was added

**Comment by JKillian** [comment]
`grunt-npm-command` released a new version to fix the peerDep error, so the build is passing now. I'm inclined to just let it stay for now, it's working well in our build.

**Comment by adidahiya** [comment]
meh, ok. filed follow-up #1235

**Pull request merged**